package/module.py

```diff
@@ -93,13 +93,13 @@ def start():
 	if not path.isdir(DIR_LINKS):
 		# no link repertory, let program tack list of link (1),
 		# or fill a file with links
-		print("""Repèrtoire des liens introuvable, 
+		print("""Repèrtoire des liens introuvable,
 1: laisser le programme choisir une liste...
```

Comment:
let program choose a list
fill a file with links
see doc.txt in config
your choise